starknet_py/tests/e2e/deploy/deployer_test.py

```diff
@@ -43,10 +49,14 @@ async def test_throws_when_calldata_not_provided(constructor_with_arguments_abi)
         match="Provided contract has a constructor and no arguments were provided.",
     ):
         deployer.create_contract_deployment(
-            class_hash=1234, abi=constructor_with_arguments_abi, cairo_version=0
+            class_hash=1234, abi=constructor_with_arguments_abi, cairo_version=1
         )
 
 
+@pytest.mark.skipif(
+    "--contract_dir=v1" in sys.argv,
```

Comment:
I think it is generally better to use `not in`
```suggestion
    "--contract_dir=v2" not in sys.argv,
```